app/views/judgements/new.html.erb

```diff
@@ -5,6 +5,19 @@
 
 <br>
 
+<%
+  @last_judgement_id = session["last_judgement_id"]
+  if @last_judgement_id != nil
+    begin
+      @last_judgement = Judgement.find(@last_judgement_id)
+    rescue
```

Comment:
double check we need this...

Reply:
lets see if when you unrate we explicitly clear the session, then we don't need the begin/rescus here...

Reply:
not sure ... this could take more time to investigate/check